### PR TITLE
[23] Add contact details form

### PR DESCRIPTION
**Pull request by asmega**
### Context

- https://trello.com/c/M7Jhkztn/23-m-add-contact-details-form

### Changes proposed in this pull request

- Add contact details form so data can be added to trainee

image

### Guidance to review

- Create or find a trainee
- Add their contact details
- Should see data inside database
- Or reload the page after saving and info should be displayed on the form

### CODE REVIEW

File: app/controllers/trainees/contact_details_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module Trainees
+  class ContactDetailsController < ApplicationController
+    def index
```
Comment (gpeng) [inline app/controllers/trainees/contact_details_controller.rb:3]: I think these actions should be `edit`. I was also wondering whether we should just have one `edit` action on `TraineesController` and just render the correct form for the path? I expect we'll need to treat some of these things separately in any case so no need to worry too much about it now. Just thinkin'.

File: app/views/trainees/contact_details/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<h2 class="govuk-heading-l">Contact details</h2>
+
+<%= form_with model: @trainee, data: { remote: false } do |f| %>
```
Comment (gpeng) [inline app/views/trainees/contact_details/index.html.erb:3]: This isn't rendering properly. `<form data-remote="false" action="/trainees/1" accept-charset="UTF-8" data-remote="true" method="post">` I think we can remove the `data: {remote: false}`. That gets added by default.

**Review by gpeng — CHANGES_REQUESTED** [review]

File: app/views/trainees/contact_details/index.html.erb (hunk 2 above)
Comment (asmega) [reply app/views/trainees/contact_details/index.html.erb]: nice spot. dev tools lies but the source tells the truth. I've added a possibly controversial commit that fixes the issue

File: app/controllers/trainees/contact_details_controller.rb (hunk 1 above)
Comment (asmega) [reply app/controllers/trainees/contact_details_controller.rb:3]: I'm not really fussed. it's kind of transparent. Maybe you kind change them all in one fell swoop in another PR :)

**Review by gpeng — APPROVED** [review]

File: app/webpacker/packs/application.js [hunk 3]
```diff
@@ -1,9 +1,5 @@
 import './govuk_assets_import'
 import '../styles/application.scss'
-import Rails from 'rails-ujs'
-import Turbolinks from 'turbolinks'
 import { initAll } from 'govuk-frontend'
 
-Rails.start()
-Turbolinks.start()
```
Comment (defong) [inline app/webpacker/packs/application.js:8]: What's with these changes?

**Review by defong — APPROVED** [review]

File: app/webpacker/packs/application.js (hunk 3 above)
Comment (asmega) [reply app/webpacker/packs/application.js:8]: that was the controversial commit. it removes rails ujs and turbolinks

**Comment by defong** [comment]
[[M] Add contact details form](https://trello.com/c/M7Jhkztn/23-m-add-contact-details-form)

**Pull request merged**